### PR TITLE
14.0.1

### PR DESCRIPTION
**Pull request by jpuri**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.1.0]
+### Changed
```
Comment (Gudahtt) [inline CHANGELOG.md:10]: This is a bug fix, so it should be v14.0.1 and categories as "Fixed"

Comment (jpuri) [reply CHANGELOG.md]: PR is updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.0.1]
+### Fixed
+- Request validation should not throw if verifyingContract is not defined in typed signature ([#328](https://github.com/MetaMask/eth-json-rpc-middleware/pull/328))
```
Comment (Gudahtt) [inline CHANGELOG.md:11]: Nit: Is this a better description? The field is in the message, not the message type or the signature for the message.
```suggestion
- Request validation should not throw if verifyingContract is not defined in typed message ([#328](https://github.com/MetaMask/eth-json-rpc-middleware/pull/328))
```

**Review by Gudahtt — APPROVED** [review]
LGTM! One suggested improvement to the wording

File: CHANGELOG.md [hunk 3]
```diff
@@ -196,7 +200,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `json-rpc-engine@5.3.0` ([#53](https://github.com/MetaMask/eth-json-rpc-middleware/pull/53))
 - `eth-rpc-errors@3.0.0` ([#55](https://github.com/MetaMask/eth-json-rpc-middleware/pull/55))
 
-[Unreleased]: https://github.com/MetaMask/eth-json-rpc-middleware/compare/v14.0.0...HEAD
+[Unreleased]: https://github.com/MetaMask/eth-json-rpc-middleware/compare/v14.1.0...HEAD
+[14.1.0]: https://github.com/MetaMask/eth-json-rpc-middleware/compare/v14.0.0...v14.1.0
```
Comment (Gudahtt) [inline CHANGELOG.md:204]: Lint fix:
```suggestion
[Unreleased]: https://github.com/MetaMask/eth-json-rpc-middleware/compare/v14.0.1...HEAD
[14.0.1]: https://github.com/MetaMask/eth-json-rpc-middleware/compare/v14.0.0...v14.1.0
```

Comment (Gudahtt) [inline CHANGELOG.md:204]: Lint fix:
```suggestion
[Unreleased]: https://github.com/MetaMask/eth-json-rpc-middleware/compare/v14.0.1...HEAD
[14.0.1]: https://github.com/MetaMask/eth-json-rpc-middleware/compare/v14.0.0...v14.0.1
```

**Review by Gudahtt — CHANGES_REQUESTED** [review]
One changelog lint error to fix

**Comment by Gudahtt** [comment]
>A new version of the package @metamask/eth-json-rpc-middleware (14.0.1) was published at 2024-09-04T17:57:41.163Z from [redacted].
The shasum of this package is 0129e5823cc219f5474e7f766dc9c9bbe2eb82a3.

**Pull request merged**